routes/signin.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (function() {
+  var express = require('express');
+  var router = express.Router();
+
+//Sign in Page
+  router.get('/', function(req, res){
+    res.render('signin', {layout: false});
+  })
+
+//Create Session
+  router.post('/validate', function(req, res, next) {
+	var auth_query = 'SELECT * FROM users WHERE user_name = ? AND password = ?';
+	var auth_param = [req.body.username, req.body.password];
+	var mysql = req.app.get('mysql');
+	mysql.pool.query(auth_query, auth_param, function(err, results, fields) {
+		if (err) return next(err);  
+		if (results.length) {
+			req.session.signedin = true;
+			req.session.active = true;
+			req.session.username= req.body.username;
+			req.session.user_id = results[0].user_id;
+			res.redirect('/posts'); //need to change to dashboard
+		} else {
+			res.redirect('/signin');
```

Comment:
A good future update would be to send back a flash message when login fails to let them know username, email, or password was incorrect.

Reply:
ok will perform in the coming sprint